amundsen_application/api/__init__.py

```diff
@@ -16,7 +18,11 @@ def init_routes(app: Flask) -> None:
 
 
 def index(path: str) -> Any:
-    return render_template("index.html", env=ENVIRONMENT)  # pragma: no cover
+    try:
+        return render_template("index.html", env=ENVIRONMENT)  # pragma: no cover
+    except jinja2.exceptions.TemplateNotFound as e:
+        logging.error("index.html template not found, have you built the front-end JS (npm run build in static/?")
```

Comment:
shouldn't use logging directly but do sth like `LOGGER = logging.getLogger(` and use the Logger obj

Reply:
@dorianj here's an example of what Tao mentioned https://github.com/amundsen-io/amundsenfrontendlibrary/blob/master/amundsen_application/api/v0.py#L17. 

I know we do have some older examples that are still not doing this correctly that you likely followed so we'll have to clean those up, but here's some quick explanation: https://docs.python.org/3/library/logging.html#logger-objects

Reply:
Thanks y'all! Updated.

@ttannis ironically enough in that example, the actual log line later doesn't actually use the `LOGGER` object 😂  -- I will open a separate PR to fix these instances.